src/shared-components/dropdown/desktopDropdown.tsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+import useResetFocus from 'src/utils/accessibility/useResetFocus';
+
+import OffClickWrapper from '../offClickWrapper';
+import ChevronIcon from '../../svgs/icons/chevron-icon.svg';
+import {
+  DropdownContainer,
+  dropdownInputStyle,
+  IconContainer,
+  DropdownOptionsContainer,
+  DropdownOption,
+} from './style';
+
+import { OptionType } from './index';
+
+type DesktopDropdownProps = {
+  value?: string;
+  options: OptionType[];
+  currentOption: OptionType;
+  textAlign: 'left' | 'center';
+  closeDropdown: () => void;
+  onOptionClick: (event: any) => void;
+  onSelectClick: () => void;
+  isOpen: boolean;
+  optionsContainerMaxHeight?: string;
+};
+
+const DesktopDropdown = ({
+  value,
+  options,
+  textAlign,
+  currentOption,
+  closeDropdown,
+  onOptionClick,
+  onSelectClick,
+  isOpen,
+  optionsContainerMaxHeight,
+}: DesktopDropdownProps) => {
+  const { initialFocus, resetFocus } = useResetFocus<HTMLDivElement>();
+
+  return (
+    <OffClickWrapper
+      onOffClick={closeDropdown}
+      css={css`
+        width: 100%;
+      `}
+    >
+      <DropdownContainer textAlign={textAlign}>
+        <div
+          id="select-input-box"
+          onClick={onSelectClick}
+          onKeyDown={(event: React.KeyboardEvent) => {
```

Comment:
Moving this into a separate function might make it more readable for other folks (esp. if paired with a code comment?)